docs/resource_sources.rst

```diff
@@ -23,6 +23,15 @@ To see all available public sources for gnomAD resources, use:
 
     list(GnomadPublicResourceSource)
 
+.. note::
+
+   The gnomAD project's bucket (``gs://gnomad-public-requester-pays``) is `requester pays <https://cloud.google.com/storage/docs/requester-pays>`_, meaning that charges for data access and transfer will be billed to your Google Cloud project.
+
+   Clusters must be configured to read requester pays buckets during creation. For example,
+
+   .. code-block::
```

Comment:
Thanks for fixing the formatting -- one downside of the suggestions feature. I shouldve done this locally first.